src/domain/Processors/VehicleProcessor.ts

```diff
@@ -145,15 +153,20 @@ export abstract class VehicleProcessor<T extends Vehicle> {
     }
   }
 
-  private getTechRecordToUpdate(vehiclesFromDB: T[], findFn: (...args: any[]) => boolean): T {
+  private getTechRecordToUpdate(
+    vehiclesFromDB: T[],
+    findFn: (...args: any[]) => boolean
+  ): T {
     if (vehiclesFromDB.length === 1) {
-      return vehiclesFromDB[0]
+      return vehiclesFromDB[0];
     }
-    const filteredRecords = vehiclesFromDB.filter(record => record.techRecord.find(findFn))
+    const filteredRecords = vehiclesFromDB.filter((record) =>
+      record.techRecord.find(findFn)
+    );
```

Comment:
This line, why?

Reply:
It's a reusable function @gjulien-bjss made to filter vehicle records. We both use it with a different find predicate but the functionality remains the same.

Reply:
No, that is fair. What I meant is what is the difference/gain of refactoring from this
```typescript
    const filteredRecords = vehiclesFromDB.filter(record => record.techRecord.find(findFn));
```
to this
```typescript
    const filteredRecords = vehiclesFromDB.filter((record) =>
      record.techRecord.find(findFn)
    );
```
?

Reply:
My comments below refer to the same thing.